tests/test_statespace/test_integrator.py

```diff
@@ -74,6 +74,54 @@ def integrator(self):
         return self.transition
 
 
+class TestIBMLinOps(TestLTISDE, TestIntegrator):
+
+    # Replacement for an __init__ in the pytest language. See:
+    # https://stackoverflow.com/questions/21430900/py-test-skips-test-class-if-constructor-is-defined
+    @pytest.fixture(autouse=True)
+    def _setup(
+        self,
+        some_ordint,
+        forw_impl_string_linear_gauss,
+        backw_impl_string_linear_gauss,
+    ):
+        self.some_ordint = some_ordint
+        spatialdim = 1  # make tests compatible with some_normal_rv1, etc.
+        with probnum_config(lazy_linalg=True):
+            self.transition = statespace.IBM(
+                ordint=self.some_ordint,
+                spatialdim=spatialdim,
+                forward_implementation=forw_impl_string_linear_gauss,
+                backward_implementation=backw_impl_string_linear_gauss,
+            )
+
+        self.G = lambda t: self.transition.driftmat
+        self.v = lambda t: self.transition.forcevec
+        self.L = lambda t: self.transition.dispmat
+
+        self.g = lambda t, x: self.G(t) @ x + self.v(t)
+        self.dg = lambda t, x: self.G(t)
+
+    @property
+    def integrator(self):
+        return self.transition
+
+    def test_dispersionmatrix(self):
+        expected = self.L(0.0)
+        received = self.transition.dispmatfun(0.0)
+        np.testing.assert_allclose(received.todense(), expected.todense())
+
+    def test_jacobfun(self, some_normal_rv1):
+        expected = self.dg(0.0, some_normal_rv1.mean)
+        received = self.transition.jacobfun(0.0, some_normal_rv1.mean)
+        np.testing.assert_allclose(received.todense(), expected.todense())
+
+    def test_driftmatfun(self):
+        expected = self.G(0.0)
+        received = self.transition.driftmatfun(0.0)
+        np.testing.assert_allclose(received.todense(), expected.todense())
```

Comment:
related to the comment above: I think these tests would be much slimmer if transitions would wrap every matrix into a linop, potentially into a dense one.

Reply:
I would like to handle this in a separate PR, together with some extensions and improvements to `linops`.

Reply:
okay, sounds good. Let's discuss details offline :)